Source/Core/Core/HW/DVD/DVDThread.cpp

```diff
@@ -342,20 +342,34 @@ static void FinishRead(u64 id, s64 cycles_late)
             (CoreTiming::GetTicks() - request.time_started_ticks) /
                 (SystemTimers::GetTicksPerSecond() / 1000000));
 
+  DVDInterface::DIInterruptType interrupt;
   if (buffer.size() != request.length)
   {
-    PanicAlertT("The disc could not be read (at 0x%" PRIx64 " - 0x%" PRIx64 ").",
-                request.dvd_offset, request.dvd_offset + request.length);
+    if (request.dvd_offset != 0x118280000 && request.dvd_offset != 0x1FB500000)
```

Comment:
I wonder if we should invert this `if` for sanity (if check, else real-error). I'm assuming you found those values from RE of the related IOS?

Reply:
Yeah, the values are found in IOS (in [DVDLowUnencryptedRead](https://wiibrew.org/wiki//dev/di#0x8D_DVDLowUnencryptedRead)), and also in various games.  I also have the same values here:

https://github.com/dolphin-emu/dolphin/blob/069e13f2ea96b24e49cda80feb4651ebd3f3fc19/Source/Core/Core/IOS/DI/DI.cpp#L250-L275

and they exist in current dolphin here (but are rejected in DVDInterface instead of making it to DVDThread):

https://github.com/dolphin-emu/dolphin/blob/d39555919d39357b49a7eb02f2d96b9a71bf128b/Source/Core/Core/HW/DVD/DVDInterface.cpp#L845-L873

I feel the real error case is more important since once a game starts, it's the only one that'll happen.  But it probably could still be swapped (especially if other error cases are added).